plugins/module_utils/rollback.py

```diff
@@ -0,0 +1,144 @@
+# (c) 2020, Julien Huon <@julienhuon> Institut National de l'Audiovisuel
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import copy
+
+from ansible_collections.community.kubernetes.plugins.module_utils.common import KubernetesAnsibleModule, AUTH_ARG_SPEC
+
+
+class KubernetesAnsibleRollbackModule(KubernetesAnsibleModule):
```

Comment:
I think we should move this code to module code since this is being used by module only and won't be reused by any other modules.

Reply:
Hi @Akasurde 

Thanks for the review. At first all was in the same file but I thought it would be easier to maintain by splitting it like it has been done for the k8s_scale module.

But I could put this code in the module code if you think it's a better option. :)

Regards,
Julien

Reply:
I also resolved the conflict with the master branch.